comps/guardrails/factuality/predictionguard/README.md

```diff
@@ -0,0 +1,47 @@
+# Factuality Check Prediction Guard Microservice
+
+[Prediction Guard](https://docs.predictionguard.com) Prediction Guard allows you to seamlessly integrate private, controlled, and compliant Large Language Models (LLM) functionality. In addition to providing a scalable LLM API, we enable you to prevent hallucinations, institute governance, and ensure compliance. Using Prediction Guard gives you quick and easy access to state-of-the-art LLMs. Acquire an API key by going [here](https://mailchi.mp/predictionguard/getting-started).
```

Comment:
I might want to edit this blurb a bit before we PR to OPEA. I'm guessing this appears in multiple places, so let's circle back to this before we PR.